.changes/allow-log-formatter-per-target.md

```diff
@@ -0,0 +1,6 @@
+---
+"log": "patch:enhance"
+"log-js": "patch"
```

Comment:
```suggestion
"log": "minor"
"log-js": "minor"
```

Thanks for the PR! I think this counts as a minor since it's a new feature/api (can't push to your branch myself). The code itself looks good :)